docs/api/instruments/bkprecision/bkprecision9130b.rst

```diff
@@ -0,0 +1,9 @@
+#########################
```

Comment:
The header sections (i.e. `####`) should match the title in length.
```
##################################
BK Precision 9130B DC Power Supply
##################################
```

Reply:
done